be/src/exec/olap_scan_node.cpp

```diff
@@ -689,11 +692,11 @@ Status OlapScanNode::build_scan_key() {
     return Status::OK();
 }
 
-static Status get_hints(TabletSharedPtr table, const TPaloScanRange& scan_range,
-                        int block_row_count, bool is_begin_include, bool is_end_include,
-                        const std::vector<std::unique_ptr<OlapScanRange>>& scan_key_range,
-                        std::vector<std::unique_ptr<OlapScanRange>>* sub_scan_range,
-                        RuntimeProfile* profile) {
+Status OlapScanNode::get_hints(TabletSharedPtr table, const TPaloScanRange& scan_range,
```

Comment:
why change the code to not static

Reply:
I do not like static method. I think we should use member function as much as possible. And I will use it in volapscannode. It is only defined in olap_scan_node.cpp .